src/http-ama-server.tsx

```diff
@@ -0,0 +1,126 @@
+import { Writable } from 'node:stream'
+
+import  { useState, useEffect } from 'react'
+import { renderToReadableStream } from "react-dom/server";
+
+import { setupOllama } from './setup-ollama.ts';
+// import { check } from './check.ts';
+
+let PORT = 3000;
+
+// // TODO: Improve using https://nodejs.org/api/stream.html
+// const server = http.createServer((req, res) => {
+//   void setupOllama(); /*.then((status) => {
+//     res.write(`ollama status is: ${status}`);
+//   }); */
+//   void check('retrieveUsingInMemory');
+
+//   res.writeHead(200, { 'Content-Type': 'text/plain' });
+
+//   res.end('This is a simple Node.js server.\n');
+// });
+
+// server.listen(PORT, () => {
+//   console.log(`Server is listening on port ${PORT}`);
+// });
+
+function AskMeAnything() {
+  // async supported but will block rendering of whole component
+  // As long as it is part of the component
+  // let status = await setupOllama();
+  // let [ollamaStatus, setOllamaStatus] = useState('idle')
+
+  // useEffect(() => {
+  //   let source = new globalThis.EventSource('/ollama-status');
+  //   source.onmessage = (e) => {
+  //       setOllamaStatus(e.data)
+  //   };
+  // }, [])
+
+  // void setupOllama().then(status => {
+  // Logged server side
+  //   console.log('status:', status)
+  //   setOllamaStatus(status)
+  // })
+
+  
+
+  return (
+    <div id="ollamaStatusContainer">
+      <span>Ollama status is: </span>
+      <span id="ollamaStatus">idle</span>
+    </div>
+  )
+}
+
+
+
+let server = Bun.serve({
+  port: PORT,
+  async fetch(request) {
+    let path = new URL(request.url).pathname;
+
+    if (path === '/ollama-status') {
+      let status = await setupOllama()
+      return new Response(status, {
```

Comment:
Need to update the event stream format. Refer: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format